
### Waypoints ###

waypoint_manager: {

waypoints:
  [[1.0, 2.5, -1.0, 0],
  [-1.0, 2.5, -1.0, 0],
  [-1.0, 2.5, 0, 0]],

threshold: 0.1,
cycle: False

}

### Controller Parameters

controller: {

x_dot_P: 0.5,
x_dot_I: 0.0,
x_dot_D: 0.0,

y_dot_P: 0.5,
y_dot_I: 0.0,
y_dot_D: 0.0,

z_dot_P: 1.0,
z_dot_I: 0.0,
z_dot_D: 0.0,

north_P: 1.0,
north_I: 0.01,
north_d: 0.05,

east_P: 2.0,
east_I: 0.01,
east_d: 0.05,

down_P: 1.0,
down_I: 0.8,
down_D: 0.2,

psi_P: 1.0,
psi_I: 0.03,
psi_D: 0.01,

tau: 0.05,

equilibrium_throttle: 0.6,

max_roll: 0.196,
max_pitch: 0.196,
max_yaw_rate: 0.2, #785,
max_throttle: 0.85,
max_n_dot: .4,
max_e_dot: .4,
max_d_dot: .4

}
